colin/core/config/config.py

```diff
@@ -11,14 +11,14 @@
 
 class Config(object):
 
-    def __init__(self, name=None):
+    def __init__(self, config=None):
         """
         Load config for colin.
 
-        :param name: str (name of the config file (without .json), default is "default"
+        :param config: str (name of the config file (without .json), default is "default"
         """
-        self.name = name or "default"
-        config_path = os.path.join(get_config_directory(), self.name + JSON)
+        self.name = config or "default"
```

Comment:
What about if config is a `path`, then this part of code does not make sense, right?

Reply:
You're right, bad naming.

Reply:
The property was never used. I've changed it to the local variable.